ginko/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Can we drop the semicolons for the sake of cleaner style?

Reply:
I agree and I think we should decide on a style guide / Linter to use